internal/types/inventory.go

```diff
@@ -112,30 +117,28 @@ type Statistics struct {
-	BuyPrice       int            `json:"buyPrice,omitempty"`
-	SellPrice      int            `json:"sellPrice,omitempty"`
-	Vocations      []string       `json:"vocations,omitempty"`
-	Type           string         `json:"type,omitempty"` // Type can either be `item` or `equipment`
-	Category       string         `json:"category,omitempty"`
-	Classification string         `json:"classification,omitempty"`
-	Recipe         map[string]int `json:"recipe,omitempty"`         // Recipe is a map of ingredients used to alchemize the inventory where the keys are inventory IDs and the values are the number of that inventory needed
-	LocationsFound []string       `json:"locationsFound,omitempty"` // LocationsFound represents the locations the Inventory can be found
-	DroppedBy      map[string]int `json:"droppedBy,omitempty"`      // DroppedBy is a map of monsters that drop the inventory where the keys are monster IDs and the values are the denominator (x) in the fraction 1/x representing the drop chance
-	IngredientFor  []string       `json:"ingredientFor,omitempty"`  // ingredientFor represents the Inventory recipes that this Inventory is part of
-	RequiredFor    []string       `json:"requiredFor,omitempty"`
-	CanBeUsedFor   []string       `json:"canBeUsedFor,omitempty"`
+	ID             string            `json:"id,omitempty"`
+	Title          string            `json:"title,omitempty"`
+	Description    string            `json:"description,omitempty"`
+	Statistics     Statistics        `json:"statistics,omitempty"`
+	Rarity         int               `json:"rarity,omitempty"`
+	BuyPrice       int               `json:"buyPrice,omitempty"`
+	SellPrice      int               `json:"sellPrice,omitempty"`
+	Vocations      []string          `json:"vocations,omitempty"`
+	Type           string            `json:"type,omitempty"` // Type can either be `item` or `equipment`
+	Category       string            `json:"category,omitempty"`
+	Classification string            `json:"classification,omitempty"`
+	Recipe         map[string]int    `json:"recipe,omitempty"`         // Recipe is a map of ingredients used to alchemize the inventory where the keys are inventory IDs and the values are the number of that inventory needed
+	LocationsFound []string          `json:"locationsFound,omitempty"` // LocationsFound represents the locations the Inventory can be found
+	DroppedBy      map[string]string `json:"droppedBy,omitempty"`      // DroppedBy is a map of monsters that drop the inventory where the keys are monster IDs and the values are the denominator (x) in the fraction 1/x representing the drop chance
+	IngredientFor  []string          `json:"ingredientFor,omitempty"`  // ingredientFor represents the Inventory recipes that this Inventory is part of
+	RequiredFor    []string          `json:"requiredFor,omitempty"`
+	CanBeUsedFor   []string          `json:"canBeUsedFor,omitempty"`
+	ImageSrc       string            `json:"imageSrc,omitempty"`
```

Comment:
This field is technically used in a couple of places to conditionally render an image. But at this time, not inventory items include `imageSrc` in the data files.